app/views/Main.js

```diff
@@ -41,8 +53,8 @@ export const Main = () => {
     NotificationService.configure(notification.status);
   }, [setCanTrack, notification.status]);
 
-  useEffect(() => {
-    updateStateInfo();
+  useDeepCompareEffect(() => {
```

Comment:
Why do we need this deep effect? Is this just for checking if selected authorities is different? 
I would recommend writing this more explicitly for what is triggering the effect / documenting why this is needed otherwise.

Reply:
Since we're using debouncing, the list of HAs could potentially end up the same as before any of the items were clicked. The deep compare effect is preventing redundant intersect checks from firing if the HA list is indeed the same as before.

Reply:
As the whole logic is now out of the render cycle, this is also not needed indeed.